chokidar/index.d.ts

```diff
@@ -5,20 +5,24 @@
 
 /// <reference types="node" />
 
-declare module "fs"
+declare module "chokidar"
 {
-    interface FSWatcher
+    export class FSWatcher
     {
+        constructor(options?: WatchOptions);
         add(fileDirOrGlob:string):void;
         add(filesDirsOrGlobs:Array<string>):void;
         unwatch(fileDirOrGlob:string):void;
         unwatch(filesDirsOrGlobs:Array<string>):void;
         getWatched():any;
+        on(event: 'add', fn: (path: string, stats?: fs.Stats) => void): this;
+        on(event: 'change', fn: (path: string, stats?: fs.Stats) => void): this;
+        on(event: 'unlink', fn: (path: string) => void): this;
+        on(event: 'raw', fn: (event: Event, path:string, details:any) => void): this;
```

Comment:
Hm I do not think that is intended behavior. Probably other node libraries are also affected as they neither depend on the DOM